### DISCUSSION TITLE
Allow filters to retain selection while switching view

### DISCUSSION THREAD

**Discussion by angelo-i** [nbrown02/FlowViz#84] [Ideas]
**Is your feature request related to a problem? Please describe.**
Losing team backlog analysis effectiveness when switching view, because the team selected is not reapplied on views.

**Describe the solution you'd like**
For example, while on Dashboard view and selecting a team in the filter, when switching to another view (e.g. WIP), have that same team selected

**Describe alternatives you've considered**
I implemented on my side, but it needs to be reconfigured with each new release.

**Additional context**
Is configured in PowerBI Desktop, [see MS article](https://learn.microsoft.com/en-us/power-bi/developer/visuals/enable-sync-slicers)

**Comment by nbrown02** [comment]
Hey Angelo - thanks for submitting this. 

Totally understand the request - however because it's been the current way for so long I'm reluctant to change it unless there is overwhelming demand for it. 

Rather than close this, I'm going to move it to a discussion to give people the chance to upvote on this idea :)

**Reply by angelo-i** [reply]
Thanks for the followup Nick! 👍
